src/types/errorsTypes.ts

```diff
@@ -0,0 +1,11 @@
+import { AxiosResponse } from "axios";
+
+export type HttpErrorType = AxiosResponse<GeneralApiError | ValidationApiError>;
```

Comment:
In this file, we can define the API error response schemas and pass it to the AxiosResopnse generic variable as a union type.